markymark/Classes/Layout Builders/UIView/Block Builders/Block/CodeViewLayoutBlockBuilder.swift

```diff
@@ -14,16 +14,29 @@ class CodeViewLayoutBlockBuilder: InlineAttributedStringViewLayoutBlockBuilder {
         return CodeBlockMarkDownItem.self
     }
 
-    override func build(_ markDownItem: MarkDownItem, asPartOfConverter converter: MarkDownConverter<UIView>, styling: ItemStyling) -> UIView {
+    override func build(
+        _ markDownItem: MarkDownItem,
+        asPartOfConverter converter: MarkDownConverter<UIView>,
+        styling: ItemStyling,
+        renderContext: RenderContext
+    ) -> UIView {
         let codeBlockMarkDownItem = markDownItem as! CodeBlockMarkDownItem
 
+        let stylingRule = (styling as? BaseFontStylingRule)
+
         let label = AttributedInteractiveLabel()
         label.numberOfLines = 0
         label.text = codeBlockMarkDownItem.content
-        label.font = (styling as? BaseFontStylingRule)?.baseFont
+
+        if let textStyle = stylingRule?.textStyle, renderContext.hasScalableFonts == true {
+            label.font = label.font.scaledFont(textStyle: textStyle)
```

Comment:
Shouldn't there be an else statement container this? Reason for question is that I'm not seeing the font being set anymore if the if-statement isn't executed

`label.font = (styling as? BaseFontStylingRule)?.baseFont`

Reply:
Thanks! good point. I was supposed to keep this line instead of deleting it:

```
label.font = (styling as? BaseFontStylingRule)?.baseFont
```